CHANGELOG.md

```diff
@@ -33,6 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
+- Added: New Missile Launcher model for Prime, Echoes, and AM2R multiworld pickups
+- Added: New Super Missile Expansion model for AM2R multiworld pickups
+- Fixed: Text is patched in all languages, not just English. 
```

Comment:
> Use hash + layout UUID check

From the ODR changelog should also be mentioned. The human readable tldr, is that if you try to play different worlds from the same session (i.e if you have dread-starter and dread-expert), that the game now will also tell you that the saves are incompatible with each other


> Fix wide shields breaking from wave 

This should also be mentioned.